src/collision.ts

```diff
@@ -12,13 +12,13 @@ export class Collision {
 	}
 
 	public static is_collision_entity_tile(entity: Entity, tile: Tile): boolean {
-		return Collision.is_collision(entity.pos_x, entity.pos_y, entity.pos_x + entity.width, entity.pos_y + entity.height,
-			tile.pos_x, tile.pos_y, tile.pos_x + tile.width, tile.pos_y + tile.height);
+		return Collision.is_collision(entity.pos.x, entity.pos.y, entity.pos.x + entity.width, entity.pos.y + entity.height,
+			tile.pos.x, tile.pos.y, tile.pos.x + tile.width, tile.pos.y + tile.height);
 	}
 
-	public static is_collision_nextpos_entity_tile(next_position: Position, entity: Entity, tile: Tile): boolean {
-		return Collision.is_collision(next_position.pos_x, next_position.pos_y, next_position.pos_x + entity.width, next_position.pos_y + entity.height,
-			tile.pos_x, tile.pos_y, tile.pos_x + tile.width, tile.pos_y + tile.height);
+	public static is_collision_nextpos_entity_tile(next_position: Point, entity: Entity, tile: Tile): boolean {
```

Comment:
Faudra peut-être revoir le nommage, ça me semble pas très clair

Reply:
Faut voir... Là c'est "is collision de la next position de l'entity par rapport à la tile"